life_expectancy/tests/conftest.py

```diff
@@ -19,5 +19,40 @@ def run_before_and_after_tests() -> None:
 
 @pytest.fixture(scope="session")
 def pt_life_expectancy_expected() -> pd.DataFrame:
+    """Fixture to load the expected output of the cleaning script."""
+
+    expected_df = pd.read_csv(
+        FIXTURES_DIR / "pt_life_expectancy_expected.csv")
+
+    expected_df = (
+        expected_df.
+        fillna(0).
+        astype({"year":int, "value": float})
+    )
+
+    return expected_df
+
+@pytest.fixture(scope="session")
+def pt_life_expectancy_input() -> pd.DataFrame:
+    """Fixture to load and clean the original dataframe
+    and only keep the first 5 rows."""
+
+    created_df = load_data()
```

Comment:
Load it with pandas, not the package's function.

Reply:
Changed 👍